### PR TITLE
Add EUPS config

### PR DESCRIPTION
**Pull request by JeremyMcCormick**

### CODE REVIEW

File: ups/sdm_schemas.table [hunk 1]
```diff
@@ -0,0 +1,4 @@
+setupRequired(sconsUtils)
+
+envPrepend(PATH, ${PRODUCT_DIR}/bin)
```
Comment (ktlim) [inline ups/sdm_schemas.table:3]: You have no `bin`, so this is not needed.

File: ups/eupspkg.cfg.sh [hunk 2]
```diff
@@ -1,5 +1,6 @@
 install()
 {
+    default_install
     mkdir -p "$PREFIX"
```
Comment (ktlim) [inline ups/eupspkg.cfg.sh:4]: I think you theoretically should be OK with removing this entire file.  But I'm not exactly sure what's going on with the installation.

Comment (JeremyMcCormick) [reply ups/eupspkg.cfg.sh:4]: > I think you theoretically should be OK with removing this entire file. But I'm not exactly sure what's going on with the installation.

This is needed for copying the `yml` directory, which won't be done by default.

**Comment by JeremyMcCormick** [comment]
I'm moving this to a ticket branch for [DM-47069](https://rubinobs.atlassian.net/browse/DM-47069).

**Pull request closed without merging**